app/src/main/res/values-pl

```diff
@@ -0,0 +1 @@
+
```

Comment:
<resources>
    <!--Universal-->
    <string name="app_name" translatable="false">Magisk Manager</string>
    <string name="magisk" translatable="false">Magisk</string>

    <!--Welcome Activity-->
    <string name="navigation_drawer_open">Otwórz szufladę nawigacji</string>
    <string name="navigation_drawer_close">Zamknij szufladę aplikacji</string>
    <string name="magiskhide">Magisk Hide</string>
    <string name="modules">Moduły</string>
    <string name="downloads">Pobieranie</string>
    <string name="log">Log</string>
    <string name="settings">Ustawienia</string>
    <string name="status">Status</string>
    <string name="install">Instalacja</string>

    <!--Status Fragment-->
    <string name="magisk_version">Zainstalowany Magisk v%1$s</string>
    <string name="magisk_version_error">Magisk nie jest zainstalowany</string>

    <string name="checking_for_updates">Sprawdzanie aktualizacji…</string>
    <string name="magisk_update_available">Magisk v%1$.1f dostępny!</string>
    <string name="cannot_check_updates">Nie można sprawdzić dostępność aktualizacji, brak internetu?</string>
    <string name="up_to_date">Zainstalowana najnowsza wersja %1$s</string>
    <string name="root_error">Zrotowany ale bez dostępu do roota?</string>
    <string name="not_rooted">Nie zrootowany</string>
    <string name="proper_root">Prawidłowo zrootowany</string>
    <string name="safetyNet_check_text">Dotknij aby sprawdzić SafetyNet</string>
    <string name="checking_safetyNet_status">Sprawdzanie statusu SafetyNet…</string>
    <string name="safetyNet_connection_failed">Nie można połączyć się z Google API</string>
    <string name="safetyNet_connection_suspended">Połączenie z Google API zostało zawieszone</string>
    <string name="safetyNet_error">Nie można sprawdzić SafetyNet bez Internetu</string>
    <string name="safetyNet_fail">SafetyNet Failed: CTS profile mismatch</string>
    <string name="safetyNet_pass">SafetyNet Passed</string>
    <string name="root_info_warning">Funkcjonalność znacznie ograniczona</string>

    <!--Install Fragment-->
    <string name="auto_detect">"(Auto) %1$s"</string>
    <string name="boot_image_title">Lokalizacja Boot Image</string>
    <string name="detect_button">Wykryj</string>
    <string name="advanced_settings_title">Zaawansowane Ustawienia</string>
    <string name="keep_force_encryption">Keep force encryption</string>
    <string name="keep_dm_verity">Keep dm-verity</string>
    <string name="current_magisk_title">Zainstalowana Wersja Magisk: v%1$s</string>
    <string name="install_magisk_title">Ostatnia Wersja Magisk: v%1$.1f</string>
    <string name="magiskify" translatable="false">Magiskify</string>

    <!--Module Fragment-->
    <string name="no_info_provided">(Nie umieszczono informacji)</string>
    <string name="no_modules_found">Nie znaleziono modułów</string>
    <string name="update_file_created">Moduł zostanie zaktualizowany przy następnym restarcie</string>
    <string name="remove_file_created">Moduł zostanie usunięty przy następnym uruchomieniu</string>
    <string name="remove_file_deleted">Moduł nie zostanie usunięty podczas następnego restartu</string>
    <string name="disable_file_created">Moduł zostanie wyłączony przy następnym restarcie</string>
    <string name="disable_file_removed">Moduł zostanie włączony przy następnym restarcie</string>
    <string name="author">Stworzone przez %1$s</string>
    <string name="fab_flash_zip">Zflashuj Moduł Zip</string>

    <!--Repo Fragment-->
    <string name="update_available">Aktualizacja jest dostępna</string>
    <string name="installed">Zainstalowany</string>
    <string name="not_installed">Nie zainstalowany</string>
    <string name="changelog">Zmiany</string>

    <!--Log Fragment-->
    <string name="menuSaveToSd">Zapisz na SD</string>
    <string name="menuSend">Wyślij</string>
    <string name="menuReload">Załaduj ponownie</string>
    <string name="menuClearLog">Wyczyść Log</string>
    <string name="logs_cleared">Log wyczyszczony</string>
    <string name="log_is_empty">Log jest pusty</string>
    <string name="logs_save_failed">Nie można zapisać logu na karcie SD:</string>

    <!--About Activity-->
    <string name="about">O Aplikacji</string>
    <string name="app_developers">Moi developerzy</string>
    <string name="app_developers_"><![CDATA[App created by <a href="https://github.com/topjohnwu">topjohnwu</a> in collaboration with <a href="https://github.com/d8ahazard">Digitalhigh</a> and <a href="https://github.com/dvdandroid">Dvdandroid</a>.]]></string>
    <string name="app_changelog">Zmiany w Aplikacji</string>
    <string name="translators" />
    <string name="app_version">Wersja Aplikacji</string>
    <string name="app_source_code">Source code</string>
    <string name="donation">Dotacja</string>
    <string name="app_translators">Tłumacze Aplikacji</string>
    <string name="support_thread">Strona Wsparcia</string>

    <!--Toasts, Dialogs-->
    
    
    <!--Settings Activity -->
    <string name="settings_general_category">Ogólne</string>
    <string name="settings_theme_title">Motyw</string>
    <string name="settings_theme_summary">Wybierz motyw</string>
    <string name="theme_default">Domyślny</string>
    <string name="theme_dark">Ciemny</string>
    <string name="settings_clear_cache_title">Wyczyść Cache Repozytorium</string>
    <string name="settings_clear_cache_summary">Clear the cached information for online repos, forces the app to refresh online</string>

    <string name="settings_magiskhide_title">Włącz Magisk Hide</string>
    <string name="settings_magiskhide_summary">Hide Magisk from various detections</string>
    <string name="settings_busybox_title">Włącz BusyBox</string>
    <string name="settings_busybox_summary">Bind mount Magisk\'s built-in busybox to xbin</string>
    <string name="settings_hosts_title">Włącz systemless hosts</string>
    <string name="settings_hosts_summary">Wsparcie systemless dla aplikacji Adblock</string>

    <string name="settings_development_category">Development</string>
    <string name="settings_developer_logging_title">Włącz zaawansowane logowanie debugowania</string>
    <string name="settings_developer_logging_summary">Zaznacz to, aby umożliwić pełne rejestrowanie</string>
    <string name="settings_shell_logging_title">Włącz rejestrowanie poleceń powłoki debugowania</string>
    <string name="settings_shell_logging_summary">Włącz, aby włączyć rejestrowanie wszystkich poleceń powłoki i jego wyjścia</string>

    <!-- Themes -->
	
	<!--URL Templates-->
    


</resources>